### ISSUE TITLE
`binjs_decode` doesn't initialize properly

### ISSUE THREAD

**Issue by dominiccooney** [binast/binjs-ref#270]
I expected this to not crash:

````
$ git show --oneline HEAD
8bf15e35 (HEAD -> master, origin/master, origin/HEAD) Merge pull request #266 from arai-a/json-conv-wtf8

$  echo 'function f() {}' | target/release/binjs_encode advanced entropy | RUST_BACKTRACE=1 target/release/binjs_decode advanced entropy 
thread 'main' panicked at 'Cannot get grammar roots "Script"', crates/binjs_io/src/entropy/baseline.rs:194:32
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
   1: std::sys_common::backtrace::print
   2: std::panicking::default_hook::{{closure}}
   3: std::panicking::default_hook
   4: std::panicking::rust_panic_with_hook
   5: std::panicking::continue_panic_fmt
   6: std::panicking::begin_panic_fmt
   7: binjs_io::entropy::baseline::BaselineDictionaryBuilder::start::{{closure}}
   8: binjs_io::entropy::baseline::build
   9: binjs_io::entropy::Options::new
  10: <binjs_io::entropy::FormatProvider as binjs_io::FormatProvider>::handle_subcommand
  11: binjs_io::Format::from_matches
  12: binjs_decode::main
  13: std::rt::lang_start::{{closure}}
  14: std::panicking::try::do_call
  15: __rust_maybe_catch_panic
  16: std::rt::lang_start_internal
  17: main
  18: __libc_start_main
  19: _start
````

**Comment by Yoric** [comment]
Apparently, for some reason, the baseline dictionary has difficulties initializing in the decoder (but not in the encoder). It's not the first time we have this kind of issue and we don't catch them because `test_*` only use a single process.

@dominiccooney Is there any chance you could turn your one-liner into a test?

**Comment by Yoric** [comment]
@dominiccooney This should be fixed by 068f02d, if you want to test.
